### ISSUE TITLE
Confirm that GO has no concern about the OMIM CURIE/identifier change to "MIM"

### ISSUE THREAD

**Issue by kltm** [geneontology/go-site#2238]
I've been asked to provide feedback on OMIM changing its CURIEs to "MIM".

Looking at the primary GO upstream data (GAFs and GPADs), there does not appear to be any OMIM data that would need to be cleansed. I.e. `reset && zgrep -i 'omim:' *-src.*.gz` returns nothing.

Looking at our own curation store (GO-CAMs), there is a single reference in a single model (http://noctua.geneontology.org/editor/graph/gomodel:55caf38600000003), which is an "experimental" model, unpublished, and almost a decade old. I.e. `reset && grep -i 'omim:' *.ttl` returns one thing.

Looking at our metadata, we have OMIM listed, but that could likely just be updated (https://github.com/geneontology/go-site/blob/f12f4d0988a91a6d6e636964de65a36e603d857f/metadata/db-xrefs.yaml#L1765).

Overall, I do not believe there would be any noticeable effect on the GO.

**Comment by kltm** [comment]
I don't believe we have any real reason to care one way or another about this. Tagging @cmungall and @pgaudet .

**Comment by balhoff** [comment]
There seem to be 4 definition xrefs in the ontology using `OMIM:`.

**Comment by kltm** [comment]
@balhoff Ah, that's worth keeping an eye on. (I'm not sure why I blanked on _the ontology_ when grepping...)

**Comment by kltm** [comment]
Changing the issue slightly as there should be some action if/when this goes forward.

**Comment by kltm** [comment]
@sierra-moxon Is there an upstream issue to watch here?

**Comment by balhoff** [comment]
> @balhoff Ah, that's worth keeping an eye on. (I'm not sure why I blanked on _the ontology_ when grepping...) Still, a relatively easy "fix" with little propagation, right?

Very easy update.

**Comment by sierra-moxon** [comment]
At the moment, bioregistry requires the prefix to be "omim" -- the discussion to change that in bioregistry is here: https://github.com/biopragmatics/bioregistry/issues/497 

In other groups trying to be compliant with bioregistry prefix declarations, waiting until it changes in bioregistry is important.   But, from the generators of the identifiers and owners of that namespace, they do seem to make it clear that want MIM to be the prefix.

**Comment by cmungall** [comment]
The ontology xrefs should be easy to fix and of course we'd keep OMIM as a synonym.

IMO OMIMs would be a good source of IMPs especially for biochemical phenotypes but looks like we get most of this in human via IEA anyway

**Comment by pgaudet** [comment]
OK for me either way.

**Comment by pgaudet** [comment]
@cmungall Should we delete this model? http://noctua.geneontology.org/editor/graph/gomodel:55caf38600000003 ? (created by you)
